tests/queries/0_stateless/01889_sqlite_read_write.sql

```diff
@@ -0,0 +1,14 @@
+DROP DATABASE IF EXISTS sqlite_database;
+
+CREATE DATABASE sqlite_database
+ENGINE = SQLite('test.sqlite3');
```

Comment:
You can put a file with sqlite data in `tests/queries/0_stateless/data_sqlite` directory. 
Then access it like `$CUR_DIR/data_sqlte/test.sqlite`. But it will require `.sh` instead of `.sql` test ([example](https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/0_stateless/00900_orc_load.sh#L7)).
So there will be commands like:
 `${CLICKHOUSE_CLIENT} --query="CREATE DATABASE sqlite_database ENGINE = SQLite(''$CUR_DIR/data_sqlte/test.sqlite');"`
...